charts/seed-bootstrap/charts/elastic-kibana-curator/values.yaml

```diff
@@ -65,6 +67,43 @@ elasticsearch:
       port: http
     initialDelaySeconds: 20
     timeoutSeconds: 5
+  metricsExporter:
+    exporter:
+      capabilities:
+        drop:
+          - SETPCAP
+          - MKNOD
+          - AUDIT_WRITE
+          - CHOWN
+          - NET_RAW
+          - DAC_OVERRIDE
+          - FOWNER
+          - FSETID
+          - KILL
+          - SETGID
+          - SETUID
+          - NET_BIND_SERVICE
+          - SYS_CHROOT
+          - SETFCAP
+    resources:
+      limits:
+        cpu: 2m
```

Comment:
Will 2m be enough? Will the amount of metrics exposed be about the same across all controlplanes?

Reply:
On my cluster the actual CPU usage is 0.1-0.2m. I think it will be similar to any cluster.
If you are still worrying about this value, i can increase it.

Reply:
I can increase it to 20m